src/Npgsql.EntityFrameworkCore.PostgreSQL/Migrations/Internal/NpgsqlHistoryRepository.cs

```diff
@@ -85,22 +85,33 @@ protected override string ExistsSql
 
         public override string GetCreateIfNotExistsScript()
         {
-            return GetCreateScript();
+            var script = GetCreateScript();
+            return script.Insert(script.IndexOf("CREATE TABLE", StringComparison.Ordinal) + 12, " IF NOT EXISTS");
         }
 
         public override string GetBeginIfNotExistsScript(string migrationId)
```

Comment:
I'm not so sure that folding all that into one statement is such a good idea, but okay.